src/bin/pg_autoctl/pgsetup.h

```diff
@@ -113,6 +113,8 @@ typedef struct pg_setup
 	PostgresControlData control;            /* pg_controldata pgdata */
 	PostgresPIDFile pidFile;                /* postmaster.pid information */
 	PgInstanceKind pgKind;					/* standalone/coordinator/worker */
+	int candidate_priority;					/* promotion candidate priority 0-100, 0 -> not a promotion candidate */
+	bool replication_quorum;				/* true if participates in write quorum, false otherwise */
```

Comment:
We should have those in a separate structure for clarity. Also, as mentioned in another comment in this review, we might add more settings to the nodes and formations later.

Reply:
separated

Reply:
Well the values are still inline at the top-level of the pg_setup structure. What I meant is that we would have a  member of the struct using the new typedef `NodeReplicationSettings`. Then we would use `pgSetup.settings.candidate_priority`.

``` C
    NodeReplicationSettings settings;
```